src/shared/content.http.ts

```diff
@@ -81,6 +81,7 @@ export default class ChromiumContentPostRoute extends BrowserHTTPRoute {
     _logger: Logger,
```

Comment:
I'd change this to `logger` as the `_` indicates that the value isn't being used here, which it is now after this PR

Reply:
done!